config/routes.rb

```diff
@@ -1,5 +1,5 @@
 if ShopifyApp.configuration.routes_enabled?
-  ShopifyApp::Engine.routes.draw do
+  Rails.application.routes.draw do
```

Comment:
I kinda like that, or make the default config for `routes_enabled` false so they have to opt-in?